### DISCUSSION TITLE
Teleamazonas

### DISCUSSION THREAD

**Discussion by MrGmaw** [iptv-org/community#285] [Show and tell]
### Channel Name

Teleamazonas

### Channel ID

Teleamazonas.ec

### Country

Ecuador

### Language

Spanish

### Website

https://www.teleamazonas.com/teleamazonas-en-vivo/

### Notes

wiki info
https://en.wikipedia.org/wiki/Teleamazonas

**Comment by what-a** [comment]
Logo:
https://upload.wikimedia.org/wikipedia/commons/thumb/e/e0/Teleamazonas_Logo.png/320px-Teleamazonas_Logo.png

**Comment by sguinetti** [comment]
Livestream: https://livestream.com/accounts/1359588/events/9942377/player

**Comment by AntiPontifex** [comment]
Link does not work: please close.
